packages/keyboard-shortcuts/src/components/shortcut-provider.js

```diff
@@ -24,14 +24,22 @@ export function ShortcutProvider( props ) {
 		if ( props.onKeyDown ) props.onKeyDown( event );
 
 		for ( const keyboardShortcut of keyboardShortcuts.current ) {
-			keyboardShortcut( event );
+			keyboardShortcut.onKeyDown( event );
+		}
+	}
+
+	function onKeyUp( event ) {
+		if ( props.onKeyUp ) props.onKeyUp( event );
```

Comment:
```suggestion
		props.onKeyUp?.( event );
```

Maybe we could use optional chaining here?